README.md

```diff
@@ -61,3 +61,492 @@ julia> logpdf_with_trans(dist, x, false) # ignoring the transformation
 julia> logpdf_with_trans(dist, x, true) # considering the transformation
 -0.7760422307471244
 ```
+
+## `Bijector` interface
+Other than the `logpdf_with_trans` methods, the package also provides a more composable interface through the `Bijector` types. Consider for example the one from above with `Beta(2, 2)`.
+
+```julia
+julia> using Bijectors; using Bijectors: Logit
+
+julia> dist = Beta(2, 2)
+Beta{Float64}(α=2.0, β=2.0)
+
+julia> x = rand(dist)
+0.7173326646959575
+
+julia> b = bijector(dist) # bijection (0, 1) → ℝ
+Logit{Float64}(0.0, 1.0)
+
+julia> y = b(x)
+0.9312689879144197
+```
+
+In this case we see that `bijector(d::Distribution)` returns the corresponding constrained-to-unconstrained bijection for `Beta`, which indeed is a `Logit` with `a = 0.0` and `b = 1.0`. The resulting `Logit <: Bijector` has a method `(b::Logit)(x)` defined, allowing us to call it just like any other function. Comparing with the above example, `b(x) == link(dist, x)`. Just to convince ourselves:
+
+```julia
+julia> b(x) == link(dist, x)
+true
+```
+
+What about `invlink`?
+
+```julia
+julia> b⁻¹ = inv(b)
+Inversed{Logit{Float64}}(Logit{Float64}(0.0, 1.0))
+
+julia> b⁻¹(y)
+0.7173326646959575
+
+julia> b⁻¹(y) == invlink(dist, y)
+true
+```
+
+Pretty neat, huh? `Inversed{Logit}` is also a `Bijector` where we've defined `(ib::Inversed{<:Logit})(y)` as the inverse transformation of `(b::Logit)(x)`. Note that it's not always the case that `inv(b) isa Inversed`, e.g. the inverse of `Exp` is simply `Log` so `inv(Exp()) isa Log` is true. Also, we can _compose_ bijectors:
```

Comment:
Add a line break before `Also, we can _compose_ bijectors:`

Also consider add subtitles like "Basic usage", "Composition", etc.

Reply:
Added a bit more structure / sections; I agree, easier to get an overview now :+1: